npm/design-system/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    /* Basic Options */
```

Comment:
These comments are invalid JSON. I realize they may be helpful, but I think `tsconfig.json` is common enough that they are unnecessary.

Reply:
They embed those comments on tsc init so I think they're fine. I also am annoyed at non-JSON JSON, but... if tsc is doing it to themselves... 🤷🏻